contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -37,6 +37,9 @@ LANGUAGE plpgsql;
  * So make sure that any SQL statement (DDL/DML) being added here can be executed multiple times without affecting
  * final behaviour.
  */
+
+ALTER TABLE sys.babelfish_authid_login_ext ADD COLUMN IF NOT EXISTS orig_loginname SYS.NVARCHAR(128) DEFAULT NULL;
```

Comment:
default NULL might not be correct, as we introduce this, we may need to refer to orig name in all of our t-sql system view from now on. default value should be current name at least

Reply:
Done! Made it NOT NULL. For already existing login during upgrade, we will copy the values from rolanme to orig_loginame.